java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture1/tasks/_10/Number10.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture1.tasks._10;
+
+import java.util.Scanner;
+import java.util.Random;
+
+/**
+ * Write a program that simulates the guess a number game.
+ * The program should generate a random number between 1 and N (N should be read from the input)
+ * and the user needs to guess it. If the user enters a number less than guessed,
+ * the program needs to output “Larger”.
+ * If the user enters a number greater than guessed - the program should output “Smaller”.
+ * If the user guesses the number correctly - the program should output “That’s it!” and finish.
+ */
+
+public class Number10 {
+    public static void main(String[] args) {
+        Number10 number10 = new Number10();
+        number10.quessGame();
+    }
+
+    private void quessGame() {
```

Comment:
I guess the method should be public in such case since it needs to provide some functionality of a class to the outer world